docs/asyncio_api.rst

```diff
@@ -4,47 +4,44 @@
 asyncio API
 ===========
 
-Create an instance of the ``AsyncClient`` to use the asyncio client API.
+Create an instance of the ``AsyncZyteAPI`` to use the asyncio client API.
 You can use the method ``request_raw`` to perform individual requests:
```

Comment:
@Gallaecio just noticed that this is referencing the old method name

Reply:
Will fix in a follow-up PR. There are also a few parts in the code that still point to AsyncClient I think.